hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -273,15 +368,50 @@ def publish_files(self, task_settings, item, publish_path):
             raise Exception("Failed to export Geometry: %s" % e)
 
         self.logger.debug(
-            "Exported group %s to '%s'." % (item.properties.fields["node"], publish_path)
+            "Exported group %s to Temporary File > '%s'." % (item.properties.fields["node"],
+                                                             publish_path)
         )
 
-        # Renaming top group name to be the asset name, in exported alembic.
-        asset_name = item.context.entity["name"]
-        self._rename_abc_top_group(publish_path_temp, str(publish_path), asset_name)
-        publish_file_temp.close()
+    def _export_gpu_abc_cache(self, task_settings, item, publish_path):
+        """
+        This method is capable of exporting the scene in a gpu alembic cache.
 
-        return [publish_path]
+        :param task_settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        :param publish_path: The output path to export files to
+
+        """
+        # find the animated frame range to use:
+        start_frame, end_frame = _find_scene_animation_range()
+        export_uv = task_settings["Export UVs"].value
+
+        try:
+            # ensure the publish folder exists:
+            publish_folder = os.path.dirname(publish_path)
+            ensure_folder_exists(publish_folder)
```

Comment:
Isn't this already done in the calling method?

Reply:
This publish_path is different from the publish_path in outerscope. 
The publish path in the calling method is the shotgun publish_path, while the one in this method is the "/var/tmp/", where we are exporting the temporary alembic, 

Maybe i should change the name within this method to "export_path"? to avoid the confusion.